.github/workflows/test-kfp-transform.template

```diff
@@ -0,0 +1,124 @@
+#
+# DO NOT EDIT THIS FILE: it is generated from test-transform.template,  Edit there and run make to change these files
+#
+name: Test KFP - @TARGET_TRANSFORM_DIR@
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - "dev"
+            - "releases/**"
+        tags:
+            - "*"
+        paths:
+            - "@TARGET_TRANSFORM_DIR@/**"
```

Comment:
I thought we wanted to make these run ONLY when @TARGET_TRANSFORM_DIR@/kfp_ray changed?

Reply:
if that is the case, then we don't think we need a special template and we can reinstate `test-src` target in kfp_ray/Makefile.

Reply:
ok